### PR TITLE
Change intro slides

### PR DESCRIPTION
**Pull request by mueller-ma**
The user will see an error message if discovery failed, so I don't see
the point any longer why we should tell him about it before.

Signed-off-by: mueller-ma <mueller-ma@users.noreply.github.com>

### CODE REVIEW

**Comment by mueller-ma** [comment]
@maniac103 May I ask for a review?

**Comment by maniac103** [comment]
This leaves us with only the themes page ... is it really worth having the intro at all just for that single item?

**Comment by mueller-ma** [comment]
> This leaves us with only the themes page

That's true, I'll add some slide. Voice recognition and nfc?

**Comment by maniac103** [comment]
I'd rather use some (partial) screenshots. For NFC a screenshot of the NFC action dialog with widget page in background with some text along the lines of 'Widget control can be bound to NFC tags by long-pressing the widget' (maybe replace 'widget' by 'item'). For voice I'd think a split screenshot like on the themes page makes sense: one half showing a part of the activity with the voice icon in the toolbar highlighted, the other half with a screenshot of the app widget.

**Comment by mueller-ma** [comment]
Not sure if I shouldn't use full screen screenshots instead.

File: mobile/src/main/java/org/openhab/habdroid/ui/IntroActivity.java [hunk 1]
```diff
@@ -68,13 +71,13 @@ public void onDonePressed(Fragment currentFragment) {
...
                             @DrawableRes int imageDrawable) {
             addSlide(AppIntroFragment.newInstance(getString(title),
-                    null,       /* Title font: null => default */
-                    getString(description), /* Description */
-                    null,       /* Description font: null => default */
-                    imageDrawable,          /* Image */
-                    colorGrey,              /* Background color */
-                    colorOpenHABOrange,     /* Title color */
-                    colorOpenHABOrange));   /* Description color */
+                    null,                                                    /* Title font: null => default */
```
[the remark this answers is not in the document]
Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/ui/IntroActivity.java]: @maniac103 The length is caused by the alignements of the comments. Can this be considered as a case where too long lines are ok?

**Pull request merged**